wkt_to_dicomseg/main.ipynb

```diff
@@ -0,0 +1,287 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "737ff207-2554-4015-9377-dc898e536585",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import packages\n",
+    "import time\n",
+    "import json\n",
+    "from PIL import Image, ImageDraw\n",
+    "import numpy as np\n",
+    "import pydicom\n",
+    "from highdicom.content import (\n",
+    "    AlgorithmIdentificationSequence,\n",
+    "    CodedConcept\n",
+    ")\n",
+    "import highdicom as hd\n",
+    "from shapely import wkt\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "88ca7877-c5d7-4e29-8402-03a36b10bb50",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# define variables\n",
+    "wkt_input = \"example/input/wkt.json\"\n",
+    "dicom_input = \"example/input/dicom_image.dcm\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "c634273c-8a0c-412a-ada8-5dd8096966d8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def create_mask_from_geometry(geometry_data, img_width, img_height):\n",
+    "    \"\"\"Create a binary mask from geometry data.\"\"\"\n",
+    "    # Create blank canvas\n",
+    "    canvas = np.zeros((img_height, img_width))\n",
+    "    mask_image = Image.fromarray(np.uint8(canvas))\n",
+    "\n",
+    "    # Get the mask polygon\n",
+    "    mask_polygon = wkt.loads(geometry_data['wkt'])\n",
+    "\n",
+    "    # Since coordinates seem to be in the correct range already, let's skip scaling\n",
```

Comment:
I would rephrase this comment a bit. Maybe something like "Coordinates should be provided in absolute terms. Any scaling should be done earlier in the process."